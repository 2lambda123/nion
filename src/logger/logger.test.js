import { titleFormatter } from './'

describe('titleFormatter', () => {
    it('should format a regular string', () => {
        expect(titleFormatter({ type: 'yay' }, 0, 500)).toMatchSnapshot()
    })

    it('should format a symbol', () => {
        expect(
            titleFormatter(
                {
                    type: Symbol('nion/NION_API'),
                    meta: { dataKey: 'currentUser' },
                },
                0,
                500,
            ),
        ).toMatchSnapshot()
    })
})
