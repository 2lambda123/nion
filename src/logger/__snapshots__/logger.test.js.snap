// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`titleFormatter should format a regular string 1`] = `"action yay (in 500.00 ms)"`;

exports[`titleFormatter should format a symbol 1`] = `"action nion/NION_API/currentUser (in 500.00 ms)"`;
